### PR TITLE
fix spinner styles

### PR DESCRIPTION
**Pull request by enniel**
This bug happens if you use a spinner without a script. [See on codepen](http://codepen.io/enniel/pen/OROxbY)

### CODE REVIEW

**Comment by traviskaufman** [comment]
@sgomes wanna take a look at this v1 change?

**Comment by sgomes** [comment]
Thanks for the PR, @enniel! I'll take a closer look next week.

File: src/spinner/_spinner.scss [hunk 1]
```diff
@@ -185,13 +185,21 @@
 }
 
 .mdl-spinner__circle-clipper {
-  display: inline-block;
+  display: block;
```
Comment (sgomes) [inline src/spinner/_spinner.scss]: I'm worried about potential backwards-compatibility issues with this change. Is the fix below compatible with us keeping this as an `inline-block`?

Comment (westy92) [inline src/spinner/_spinner.scss]: It seems fine without it: http://codepen.io/anon/pen/vyRdwo.

Comment (enniel) [inline src/spinner/_spinner.scss]: I'm fixed them.

**Comment by sgomes** [comment]
LGTM, thanks @enniel!

**Pull request merged**